packages/client/src/generation/utils/buildNFTAnnotations.ts

```diff
@@ -95,7 +99,9 @@ function buildNFTEngineAnnotation(engineType: ClientEngineType, platform: Platfo
  * @returns
  */
 function buildNFTSchemaAnnotation(engineType: ClientEngineType, relativeOutdir: string) {
-  if (engineType === ClientEngineType.DataProxy) return ''
+  if (engineType === ClientEngineType.Library || engineType === ClientEngineType.Binary) {
```

Comment:
Isn't this condition always true now?

Reply:
Yes, it is. Thanks for that. I will fix it in the finalization PR.

Reply:
(although not critical since nothing would be found)